client/rest-high-level/build.gradle

```diff
@@ -26,6 +27,9 @@ apply plugin: 'nebula.maven-scm'
 group = 'org.elasticsearch.client'
 archivesBaseName = 'elasticsearch-rest-high-level-client'
 
+Task copyRestSpec = RestIntegTestTask.createCopyRestSpecTask(project, false)
+test.dependsOn(copyRestSpec)
```

Comment:
these build changes are temporary, I just hacked to make the new test work. The new test needs the spec in its classpath, yet it is not an integ test. Suggestions are welcome on how to improve this and make this cleaner.